src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -83,4 +84,12 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    
+    default String getPrintableString(Printable... printable) {
```

Comment:
consider adding a header comment for this method; the behavior of this is different from the one in the `Printable` interface.